lib/stripEndings.js

```diff
@@ -0,0 +1,52 @@
+const toUnicode = require( './toUnicode' )
+const toAscii = require( './toAscii' )
+const { getRegexClass, getRegexGroup } = require( './regex-utils' )
+
+// Line endings in both ASCII, Unicode, and English
+const endingClass = getRegexClass( [ '।', '॥', ']', '[', '|' ] )
+// Sometimes translation line endings begin with these characters, before numbers
+const optionalEndingClass = getRegexClass( [ '(' ] )
+// Remove any broken endings
+const brokenEndingClass = getRegexGroup( [ '()' ] )
+
+// All numbers in ASCII, Unicode
+const numberClass = getRegexClass( [
+  ...Array.from( { length: 10 }, ( _, i ) => i.toString() ),
+  ...Array.from( { length: 10 }, ( _, i ) => toUnicode( i.toString() ) ),
+] )
+
+// Rahao in English, ASCII, Unicode
+const pauseGroup = getRegexGroup( [ 'ਰਹਾਉ', toAscii( 'ਰਹਾਉ' ), 'Pause' ] )
+
+const matchers = [
+  // Endings followed by any number => match the rest of the line
+  ` ?(${endingClass}|${optionalEndingClass}?)${numberClass}.*`,
```

Comment:
So you could have `mwJ mhlw 5 Gru 1 ]` and it would capture all of `5 Gru 1 ]`? Since we don't want to use this on sirlekhs / headers, that's okay. I couldn't find a scenario where it broke content. 

However, it should be clear that this function should not be used on headers/sirlekhs. Here are some examples in vs code with regex `("Sirlekh",\n.+"gurmukhi": \{\n.+": ")(.* \d.*)("\n.*\},)`

```
sRI muKvwk pwiqswhI 10 ]
so dru rwgu Awsw mhlw 1 ]
rwgu Awsw mhlw 4 so purKu ]
rwgu isrIrwgu mhlw pihlw 1 Gru 1 ]
isrIrwgu mhlw 1 Gru dUjw 2 ]
isrIrwgu mhlw 1 Gru 1 AstpdIAw ]
mÚ 3 ]
```

I'm a bit iffy on this sirlekh / header (I don't think anyone will search first letters for this, and it doesn't affect translation/translit):
```
vwr mwJ kI. qQw slok mhlw 1 mlk murId qQw cMdRhVw sohIAw kI DunI gwvxI ]
```

Perhaps we can either mention not intending to use on sirlekh/header or we could include some of these in tests to make it obvious what would happen if used in that unintended way?

Reply:
Ok. Can add a test and doc point about it. We’ll apply for non-sirlekhs at app/DB-level